README.md

```diff
@@ -1,3 +1,15 @@
+# ⛔️ DEPRECATED
+
+_This example is out of date and will not be maintained._
```

Comment:
recommend being more specific: it probably does not work, and *is* not maintained